extractor/src/producer_class.py

```diff
@@ -28,7 +28,7 @@ def _generate_config(self):
         Generate configuration dictionary for consumer
         :return:
         """
-        config = {"bootstrap.servers": self.broker}
+        config = {"bootstrap.servers": self.broker, "linger.ms": 0.5}
```

Comment:
comz

Reply:
Added. Fyi I didn't set `linger.ms` in the transformer since it appears to lower performance.